_posts/2019-05-09-How-We-Release-Code.md

```diff
@@ -0,0 +1,96 @@
+
+This blog post is an attempt at describing how we manage the process of proposing, reviewing and approving projects to become open source code while at the same time ensuring that code we release follow our compliance rules and that the maintainers of the projects are aware of the risks and responsibilities involved.
+
+[Our formal release guidelines are available here](https://opensource.zalando.com/docs/releasing/index/)
+
+## Overview
+
+The process roughly involves 5 steps which takes the project from internal source code, through a review phase to an incubation organisation, which later either results in being graduated into our main organisation or archived as an inactive project due to lack of activity or maintainers: 
+
+1. An internal project is proposed for release by a Zalando engineer
+2. The project proposal is reviewed by the internal open source review group
+3. If approved, the project is published on the Zalando Incubator on Github
+4. The project activity and health is monitored by the open source team
+5. The project graduates from the incubator and into the main Zalando organisation, or the project is decommission and marked as archived.
+
+The part where we monitor projects and decide on whether to keep them or archive them will be outlined in greater detail in a later blogpost. 
+
+## Proposing a new open source project
+First step in getting an internal project published on the Zalando Incubator is to apply for publication by filling out a google form and understanding our requirements [which are available here](https://opensource.zalando.com/docs/releasing/index/)
+
+Anyone inside of Zalando can do this and this step serves 2 purposes:
+
+- **To collect information required to publish a project**, such as its current location, who will be maintaining it and the long term plan for maintaining it. 
+- **To set expectations for the maintainers**, such as amount of time needed to maintain the project, sign-off from the developers lead and ensuring the project does not require Zalando-specific dependencies
+
+![Project Release Form](/assets/img/os-release-form.png)
+
+You can see a public version of the [approval form without validation here](https://goo.gl/forms/9C4xlel5DlIK52Xw1).
+
+Questions addressing who have signed-off on the publishing and how many hours the developers have been granted to maintain the project serve as a good way to set expectations, both to the lead who sign off and for the maintainers. To run a sustainable project requires time and resources and we do not expect developers to use their private time, instead granting work time should be part of the conversation. 
+
+We also address the need to have basic project health files in place such as Code of conduct, ways for users to get in touch in case of security issues and to reach out to the maintainers - we provide maintainers with a [boiler-plate set of files](https://github.com/zalando-incubator/new-project) for guidance. We do this for 2 reasons:
+
```

Comment:
See above comment re numbered lists (consistency).

Reply:
"""
We also address the need to have basic project health files in place such as 
a Code of Conduct, ways for users to get in touch in case of security issues, 
features or bugs, by providing maintainers with a [standaerd set of files](https://github.com/zalando-incubator/new-project). We do this for 2 reasons:
"""